Mathlib/CategoryTheory/Galois/Decomposition.lean

```diff
@@ -160,6 +167,144 @@ lemma connected_component_unique {X A B : C} [IsConnected A] [IsConnected B] (a
 
 end Decomposition
 
+section GaloisRep
+
+/-! ### Galois representative of fiber
+
+If `X` is any object, then its fiber is represented by some Galois object: There exists
+a Galois object `A` and an element `a` in the fiber of `A` such that the
+evaluation at `a` from `A ⟶ X` to `F.obj X` is bijective.
+
+To show this we consider the product `∏ (fun _ : F.obj X ↦ X)` and let `A`
+be the connected component whose fiber contains the element `a` in the fiber of the self product
+that has at each index `x : F.obj X` the element `x`.
+
+This `A` is Galois and evaluation at `a` is bijective.
+
+Reference: Lenstra, 3.14
```

Comment:
And refer to it here please.

Reply:
Please check if this is what you meant, I am not sure I interpreted the instructions in `docs/references.bib` correctly.